openhands/controller/state/state.py

```diff
@@ -156,14 +156,14 @@ def get_current_user_intent(self) -> tuple[str | None, list[str] | None]:
 
         return last_user_message, last_user_message_image_urls
 
-    def get_last_agent_message(self) -> str | None:
+    def get_last_agent_message(self) -> MessageAction | None:
```

Comment:
I think this signature change (and/or below) may require adaptations in /evaluation.

Reply:
Ahh thank you, I need to get better about searching there. Also, we should start linting it....